acropolis-service/src/main/kotlin/org/ephyra/acropolis/service/impl/ApplicationSoftwareService.kt

```diff
@@ -38,6 +38,17 @@ class ApplicationSoftwareService : IApplicationSoftwareService {
         return persistence.findByName(name, projectId)
     }
 
+    override fun findAll(projectName: String): List<ApplicationSoftwareEntity> {
```

Comment:
Do we want doc comments on the overrides?

Reply:
Only if the override has details which are not covered by the interface documentation but need to be documented